packages/dd-trace/src/llmobs/writers/evaluations.js

```diff
@@ -0,0 +1,28 @@
+'use strict'
+
+const BaseWriter = require('./base')
+
+class LLMObsEvalMetricsWriter extends BaseWriter {
+  constructor (config) {
+    super({
+      endpoint: '/api/intake/llm-obs/v1/eval-metric',
```

Comment:
(noticed this is the same as in the Python SDK), let's move this endpoint to the constants file for easier maintenance